mbtools/models.py

```diff
@@ -120,39 +141,74 @@ def delete_empty_categories(self):
                 elem.getparent().remove(elem)
 
 
+class MoodleLessonPage:
+    def __init__(self, tree, location):
+        self.tree = tree
+        self.location = location + f' (page: {self.name()})'
+        self.id = int(self.tree.attrib['id'])
+
+    def name(self):
+        return self.tree.xpath("title")[0].text
```

Comment:
Ok I'm familiarizing myself with the @property tags, and it seems like we'd want to use those in case the underlying data might be changed. Here that'd be the xml for the activity. It seems like that is unlikely to happen, I'll toss it in the constructor.